### DISCUSSION TITLE
Database Webhooks playload is empty object

### DISCUSSION THREAD

**Discussion by gaodeng** [supabase/supabase#19477] [Questions]
https://supabase.com/docs/guides/database/webhooks#payload

**Comment by steve-chavez** [comment]
This is only related to docs right?

**Comment by gaodeng** [comment]
> This is only related to docs right?

@steve-chavez 
I’m not sure。
so does that mean the webhook payload should be an empty object {}, with no information at all? If that’s the case, then the documentation is incorrect, otherwise it’s a bug.

**Comment by AdditionAddict** [comment]
Fixed by switching from GET to POST

**Comment by gregnr** [comment]
@gaodeng does @AdditionAddict's fix solve your issue?
